From 75d656e7f13b9966567c06798cb842ff89258693 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Sun, 20 May 2018 21:15:36 +1200
Subject: [PATCH] TRITON-309 - add new triton-origin-multiarch-18.1.0 image
 Reviewed by: Trent Mick <trent.mick@joyent.com> Approved by: Trent Mick
 <trent.mick@joyent.com>

---
 images.json | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/images.json b/images.json
index 71fb940..5c71a02 100644
--- a/images.json
+++ b/images.json
@@ -15,5 +15,22 @@
             "sudo-1.8.15"
         ],
         "protoMinMemoryMb": "1024"
+    },
+    {
+        "namePrefix": "triton-origin-multiarch",
+        "desc": "Triton/Manta component origin image based on multiarch 18.1.0",
+        "origin": {
+            "uuid": "1ad363ec-3b83-11e8-8521-2f68a4a34d5d",
+            "name": "minimal-multiarch",
+            "version": "18.1.0"
+        },
+        "pkgsrc": [
+            "coreutils-8.26",
+            "curl-7.59.0",
+            "gsed-4.4",
+            "patch-2.7.6",
+            "sudo-1.8.22"
+        ],
+        "protoMinMemoryMb": "1024"
     }
 ]
-- 
2.21.0

